packages/gatsby-theme-docs/gatsby-node.js

```diff
@@ -27,10 +28,13 @@ exports.onPreBootstrap = ({ reporter }) => {
   ];
   requiredDirectories.forEach((dir) => {
     if (!fs.existsSync(dir)) {
-      reporter.info(`creating the ${dir} directory`);
+      gatsbyApi.reporter.info(`creating the ${dir} directory`);
       fs.mkdirSync(dir);
     }
   });
+
+  // Load theme add-ons
+  loadThemeAddOns(gatsbyApi, themeOptions);
```

Comment:
```suggestion
  createAddOnThemeFiles(gatsbyApi, themeOptions);
```

Reply:
True. I renamed it to `bootstrap` instead of `load`, since it's also part of the `onPreBootstrap` step.

Hope it's fine for you.